src/api/goods.ts

```diff
@@ -1,7 +1,7 @@
+/* eslint-disable max-len */
```

Comment:
remove
```suggestion

```

Reply:
i need it because API_URL have a long url